components/org.wso2.carbon.identity.api.server.idp/org.wso2.carbon.identity.api.server.idp.v1/src/main/resources/idp.yaml

```diff
@@ -2388,3 +2392,6 @@ components:
       required:
         - name
         - idp
+    Service:
+      type: string
```

Comment:
here also services can be an array. Let's change it to be the same as IdentityProviderTemplateListItem

Reply:
also make it readOnly.. such that it cannot be sent via request.

Reply:
Here service is the list item for services attribute in IdentityProviderTemplateListItem. I didn't add services attribute for IdentityProviderTemplate. Since we get the full template when we retrieve templates by the id.